Tests/DatadogTests/Datadog/Core/System/MobileDeviceTests.swift

```diff
@@ -52,13 +52,25 @@ class MobileDeviceTests: XCTestCase {
         )
     }
 
-    func testWhenRunningOnMobile_itUsesProcessInfoPowerMode() {
-        XCTAssertTrue(
-            MobileDevice(uiDevice: UIDeviceMock(), processInfo: ProcessInfoMock(isLowPowerModeEnabled: true)).currentBatteryStatus().isLowPowerModeEnabled
+    func testGivenInitialLowPowerModeSettingValue_whenSettingChanges_itUpdatesIsLowPowerModeEnabledValue() {
+        // Given
+        let isLowPowerModeEnabled: Bool = .random()
+
+        let mobileDevice = MobileDevice(
+            uiDevice: UIDeviceMock(),
+            processInfo: ProcessInfoMock(isLowPowerModeEnabled: isLowPowerModeEnabled)
         )
-        XCTAssertFalse(
-            MobileDevice(uiDevice: UIDeviceMock(), processInfo: ProcessInfoMock(isLowPowerModeEnabled: false)).currentBatteryStatus().isLowPowerModeEnabled
+
+        XCTAssertEqual(mobileDevice.currentBatteryStatus().isLowPowerModeEnabled, isLowPowerModeEnabled)
+
+        // When
+        NotificationCenter.default.post(
```

Comment:
if you think we might add more test cases to `MobileDeviceTests` later, you may want to use different `NotificationCenter` instances like [this](https://github.com/DataDog/dd-sdk-ios/blob/master/Sources/Datadog/Core/System/AppStateListener.swift#L108) to avoid interference between cases.

Reply:
Good idea 👌 - added.